tests/storage/test_benchmark.py

```diff
@@ -68,7 +68,7 @@ def get_by_lines_parameters() -> list[tuple[int, str, bytes]]:
             _IS_MULTILINE_PATTERN,
             _IS_MULTILINE_WHILE,
         ]:
-            for newline in [b"", b"\n", b"\r\n"]:
```

Comment:
i'm guessing empty `newline` denotes either the document isn't multiline format, or we have reached the end of the newline delimited data? how come we're removing the case here?

Reply:
good catch, leftover from changes during debugging, I will revert it!